Assets/Scripts/Game Controllers/LevelEndTriggerController.cs

```diff
@@ -3,11 +3,35 @@
 
 public class LevelEndTriggerController : MonoBehaviour
 {
+    [SerializeField]
+    private int indexForNextScene;
+
+    [SerializeField]
```

Comment:
It seems like `[SerializeField]` doesn't do anything for `const` variables, so I think it can be removed.